### ISSUE TITLE
How to add objects to planets

### ISSUE THREAD

**Issue by JohnArctiguy** [SebLague/Solar-System#60]
I want to add objects like trees or rocks to planets, but I don't understand how I can do this. Does somebody know how to do that?

**Comment by Panicat** [comment]
Yes, you raycast down from random sphere points around the planet, and spawn a prefab there. However, for dense rocks or grass, or with big planets, this would lag. I don't know how to make a more advanced system for rocks like KSP 1's Parallax mod or KSP 2's rocks.

**Comment by JohnArctiguy** [comment]
Thanks for method!!!
